ibm_catalog.json

```diff
@@ -190,6 +199,36 @@
               }
             ]
           },
+          "iam_permissions": [
+            {
+              "service_name": "sysdig-monitor",
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Editor"
+              ]
+            },
+            {
+              "service_name": "atracker",
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Writer",
+                "crn:v1:bluemix:public:iam::::role:Editor"
```

Comment:
Again I don't see why `sysdig-monitor` or `atracker` permissions are needed here - The DA is not creating these resources